pidstat.h

```diff
@@ -3,6 +3,9 @@
  * (C) 2007-2016 by Sebastien Godard (sysstat <at> orange.fr)
  */
 
+#include <sys/param.h>
```

Comment:
Both this line, 6, and line 7 probably need supported platform protection that matches what needs to be done down on line 21.